docs: add cluster upgrade section to tutorial (#10399)
spectrocloud/librarium: docs/docs-content/tutorials/clusters/cluster-templates/standardize-clusters-with-cluster-templates-terraform.md

## Upgrade Clusters from the Template

> In this section, you will trigger an immediate upgrade of both clusters attached to `tf-cluster-template-aws`.

Reviewer: ```suggestion
In this section, trigger an immediate upgrade of both clusters attached to `tf-cluster-template-aws`.
```

> `tf-dev-cluster` and `tf-prod-cluster` will move from profile version `1.0.0` to `1.1.0`.

Reviewer: ```suggestion
`tf-dev-cluster` and `tf-prod-cluster` move from profile version `1.0.0` to `1.1.0`.
```

}
```

</TabItem>

</Tabs>

In `terraform.tfvars`, set `upgrade_now_timestamp` to the current time in RFC3339 format.

```hcl title="terraform.tfvars" hideClipboard
upgrade_now_timestamp = "2026-05-12T14:30:00Z"
```

Issue the `terraform plan` command to preview the changes.

```shell
terraform plan
```

Terraform reports one update to the cluster template resource.

```text hideClipboard title="Expected output"
# spectrocloud_cluster_config_template.aws_template[0] will be updated in-place
  ~ resource "spectrocloud_cluster_config_template" "aws_template" {
      + upgrade_now = "2026-05-12T14:30:00Z"
        ...
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

Apply the changes to trigger the upgrade.

```shell
terraform apply -auto-approve
```

```bash hideClipboard title="Expected output"
spectrocloud_cluster_config_template.aws_template[0]: Modifying...
spectrocloud_cluster_config_template.aws_template[0]: Modifications complete after 3s

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
```

The template update triggers an upgrade on both clusters attached to the template. `tf-dev-cluster` and

> `tf-prod-cluster` both begin moving from profile version `1.0.0` to `1.1.0`. The upgrade may take several minutes. In

Reviewer: ```suggestion
`tf-prod-cluster` both move from profile version `1.0.0` to `1.1.0`. The upgrade may take several minutes. In
```
